ra-tls/ra-tls-server/lib/Makefile

```diff
@@ -83,7 +83,7 @@ ifndef ECDSA
 endif
 
 App_Include_Paths := $(Wolfssl_Include_Paths) -I$(SGX_SDK)/include -I$(SGX_RA_TLS_ROOT) -I$(INCDIR) $(SGX_DCAP_INC) -I$(shell readlink -f .)
-App_C_Flags := $(SGX_COMMON_CFLAGS) -fPIE -Wno-attributes -Wall -Wno-unused-const-variable $(App_Include_Paths) $(Wolfssl_C_Extra_Flags)
+App_C_Flags := $(SGX_COMMON_CFLAGS) -fPIC -Wno-attributes -Wall -Wno-unused-const-variable $(App_Include_Paths) $(Wolfssl_C_Extra_Flags)
```

Comment:
So what happens if -fPIE is used?

Reply:
Build `libskeleton-v3.so`  will fail with the information. `relocation xxx can not be used when making a shared object; recompile with --FPIC`

Reply:
Please submit an issue and set its fix milestone to 0.7.0